.github/workflows/external-integration.yml

```diff
@@ -0,0 +1,84 @@
+name: External Integration
+
+on:
+  pull_request:
+    types: [labeled]
```

Comment:
this should trigger also for new commit, opened pr. The rule should be only if the `int:azure-specs` label is present (not just when it was added)